### ISSUE TITLE
SerializeIndex config option not working

### ISSUE THREAD

**Issue by cgarveyie** [yapapaya/jquery-cloneya#70]
Hi there, my understanding of how this is intended to function is that the name property of input fields would receive a unique identifer in the same way as the id fields do.

However, this does not appear to be working.

**Comment by plasebo** [comment]
Yes noticed that the config for serializeIndex is missing. Add it to:
`var name = "cloneya", defaults = {
   serializeIndex: true,
}
`
